vector/src/main/java/im/vector/adapters/VectorRoomDetailsMembersAdapter.java

```diff
@@ -535,24 +459,22 @@ public void run() {
                         return;
                     }
 
-                    presentMembersList.addAll(actualParticipants);
-
                     uiHandler.post(new Runnable() {
                         @Override
                         public void run() {
                             // test if the pattern has been updated while searching the items.
                             if (TextUtils.equals(mSearchPattern, fPattern)) {
                                 mDisplayNamesList = displayNamesList;
                                 mRoomMembersListByGroupPosition = roomMembersListByGroupPosition;
-                                mGroupIndexPresentMembers = -1;
-                                mGroupIndexPresentMembers = -1;
```

Comment:
This line was duplicate?

Reply:
yes :(, the second one would have be `mGroupIndexInvitedMembers`